### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by laughedelic**
A quick update for the roadmap.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +27,21 @@ still learning and exploring what's possible.
...
+* Integration with the sbt server:
+  - [x] Run custom sbt command on save
+  - [x] Receive compilation errors (`textDocument/publishDiagnostics`)
 * Linting with Scalafix (`textDocument/publishDiagnostics`):
   - [x] On build compile
   - [x] As you type
 * Refactoring with Scalafix:
   - [ ] Quick-fix inspections (`textDocument/codeAction`)
+    + [x] Remove unused imports
   - [ ] Rename local symbol (`textDocument/rename`)
```
Comment (gabro) [inline README.md]: 👆 this too!

File: README.md [hunk 2]
```diff
@@ -27,16 +27,21 @@ still learning and exploring what's possible.
...
+  - [x] Run custom sbt command on save
+  - [x] Receive compilation errors (`textDocument/publishDiagnostics`)
 * Linting with Scalafix (`textDocument/publishDiagnostics`):
   - [x] On build compile
   - [x] As you type
 * Refactoring with Scalafix:
   - [ ] Quick-fix inspections (`textDocument/codeAction`)
-  - [ ] Rename local symbol (`textDocument/rename`)
+    + [x] Remove unused imports
+  - [x] Rename local symbol (`textDocument/rename`)
```
Comment (olafurpg) [inline README.md]: strictly speaking we don't use scalafix for rename because it's such a simple text edit

File: README.md [hunk 3]
```diff
@@ -27,16 +27,21 @@ still learning and exploring what's possible.
...
   - [ ] Rename global symbol (`textDocument/rename`)
 * Formatting with Scalafmt:
   - [x] Whole file (`textDocument/formatting`)
   - [ ] Selected range (`textDocument/rangeFormatting`)
+  - [x] On save (`textDocument/willSaveWaitUntil`)
```
Comment (olafurpg) [inline README.md]: 👍

**Comment by laughedelic** [comment]
> Judging from the roadmap it would almost seem like a complete project! It would be good to write down known caveats at some point

Yeah, this roadmap needs some rewrite with clear separation between stable and experimental features. Probably it would make sense to shape it after the config.

Anyway, I just wanted to update what we already had. Merging!

**Pull request merged**